management-core/src/test/java/com/ns/task/product/ProductCoreServiceImpTest.java

```diff
@@ -0,0 +1,101 @@
+package com.ns.task.product;
```

Comment:
What would the app do if the DB throws an exception?

Reply:
This will be included in the next commit